docs/upgrade/2.10.1_to_2.11.0.rst

```diff
@@ -0,0 +1,93 @@
+.. _latest_upgrade_guide:
+
+Upgrade from 2.10.1 to 2.11.0
+=============================
+
+Preparing for the Ubuntu 24.04 (Noble) migration
+------------------------------------------------
+
+The 2.11.0 release includes a number of features that will help ensure
+your SecureDrop server is prepared for the automated migration to Ubuntu 24.04
+(Noble) in early 2025.
+
+After updating to SecureDrop 2.11.0 using the steps below, it's important
+that you review our :doc:`Ubuntu 24.04 (Noble) migration guide <../admin/maintenance/noble_migration_prep>`,
+and run the ``securedrop-noble-migration-check`` script prior to January 31st, 2025
+to ensure your server is ready for the migration.
+
+
+Update Servers to SecureDrop 2.11.0
+------------------------------------
+
+Servers running Ubuntu 20.04 will be updated to the latest version of SecureDrop
+automatically within 24 hours of the release.
+
+Update Workstations to SecureDrop 2.11.0
+----------------------------------------
+
+.. important:: We always recommend backing up your workstations prior to
```

Comment:
Should this be removed now that the Tails 6 stuff was also removed?

Reply:
I think it's generally good advice to recommend backups prior to updates (even relatively low-risk ones), so I'm inclined to leave it in

Reply:
I agree with that sentiment, but I don't think the language matches, e.g. "but we *especially* recommend it before a major Tails version bump" even though we aren't telling them to do said bump. I'll defer to you on keeping it, but maybe the language can be further tweaked?

Reply:
This was a great flag, looking at it with fresh eyes the language didn't match.

I stripped it back to a more minimal "We recommend backing up your workstations prior to any upgrades. See our backup instructions for more information." Let me know if this works for you.